config.py

```diff
@@ -65,6 +65,6 @@ class TestingConfig(Config):
     LOGGING_LEVEL = 'ERROR'
     SECURITY_USER_NAME = 'admin'
     SECURITY_USER_PASSWORD = 'secret'
-    DATABASE_URI = "sqlite:///:memory:"
+    DATABASE_URI = "postgres://postgres:postgres@localhost:6432/postgres"
```

Comment:
I think it'd be useful to still be able to set this from an environment variable when testing so they can choose which port to run their database on